internal/oplog/logged_db_test.go

```diff
@@ -66,7 +67,7 @@ func TestGobLoggedDB(t *testing.T) {
 	rwtx.Connect(u, p, db.UserProfile)
 	rwtx.Commit()
 
-	appDB2 := db.New()
+	appDB2 := db.New(db.CodeExecutor{Invoke: runtime.Invoke})
```

Comment:
this dependency seems awkward to me to specify everywhere. I wonder if we should just have db.New() and db.NewTest() that configures a mock executor or something like that.

Separately, it's a bit funny to me that db takes a CodeExector as the argument -- seems like all it needs here is the runtime.Invoke

Reply:
No problem, updated!